docs/sphinx_api_docs_source/public_api_missing_threshold.py

```diff
@@ -101,6 +101,7 @@
     "File: great_expectations/validator/validation_graph.py Name: resolve",
     "File: great_expectations/datasource/new_datasource.py Name: get_batch_list_from_batch_request",
     "File: great_expectations/datasource/fluent/config.py Name: yaml",
+    "File: great_expectations/datasource/fluent/config.py Name: pop",
```

Comment:
I don't understand why I'm having to ignore this. The `GxConfig` class should not be referenced in any docs nor should it be part of the public API.